docs/output.md

```diff
@@ -682,7 +699,7 @@ Plots will show:
 
 For all samples:
 
-**Output directory: `results/Reports/[SAMPLE]/VCFTools`**
+**Output directory: `{outdir}/Reports/[SAMPLE]/VCFTools`**
```

Comment:
```suggestion
**Output directory: `{outdir}/reports/vcftools/[sample]`**
```

Reply:
@SusiJo : I think `[sample]` should be `<sample>` in the above line. A similar comment applies many other places in the documentation.